tests/core/notification/test_events_published.py

```diff
@@ -166,6 +166,7 @@ def test_events_published_for_scenario_submission():
     register_id_0, register_queue_0 = Notifier.register()
     all_evts = RecordingConsumer(register_id_0, register_queue_0)
     all_evts.start()
+    # Write value to the unwritten data node trigger 3 is_submittable update events for the scenario, sequence and task
```

Comment:
👍🏻 
Can you also update the fiollowing comment to make is correct regarding the three new events?